File: Java(Udemy)/FrasesdoHelpDesk/app/src/main/java/fraseshelpdesk/nivaldojr/com/frasesdohelpdesk/MainActivity.java
package fraseshelpdesk.nivaldojr.com.frasesdohelpdesk;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    private TextView textoNovaFrase;
    private Button botaoNovaFrase;

    private String[] frases = {"Realmente", "É uma questão de compreensão senhor.",
            "JUB up", "Genial", "200%"};

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textoNovaFrase = findViewById(R.id.textoNovaFraseId);
        botaoNovaFrase = findViewById(R.id.botaoNovaFraseId);


        botaoNovaFrase.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Random randomico = new Random();
                int num = randomico.nextInt(frases.length);
                textoNovaFrase.setText(frases[num]);
            }
        });
    }
}

File: Kotlin(Unifavip Wyden)/Calculator/app/src/main/java/aula3/nivaldojr/com/calculator/MainActivity.kt
package aula3.nivaldojr.com.calculator

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val button = findViewById<Button>(R.id.calculate_button)

        button.setOnClickListener(){
            val editTextNumber1 = findViewById<EditText>(R.id.input_num1)
            val editTextNumber2 = findViewById<EditText>(R.id.input_num2)

            var num1 = 0.0
            if(editTextNumber1.text.toString() != ""){
                num1 = editTextNumber1.text.toString().toDouble()
            }

            var num2 = 0.0
            if(editTextNumber2.text.toString() != ""){
                num2 = editTextNumber2.text.toString().toDouble()
            }

            var result = num1 * num2
            Toast.makeText(this, "Result: $result", Toast.LENGTH_SHORT).show()

        }
    }
}
